test/controllers/call_lists_controller_test.rb

```diff
@@ -65,7 +65,7 @@ class CallListsControllerTest < ActionDispatch::IntegrationTest
       assert_no_difference '@user.call_list_entries.count' do
         patch remove_patient_path(@patient_1), xhr: true
       end
-      assert_response :success
+      assert_response :not_found
```

Comment:
Why is this flipping?

Reply:
this was a 'how did this ever work' kinda thing. I'm honestly not sure why this catches right in pg but not really in mongo, but a 404 does feel closer to a sensible response here.